content/blog/sleep/index.md

```diff
@@ -0,0 +1,78 @@
+
+`youtube: https://www.youtube.com/watch?v=1ZSzI5mHwS8`
+
+Let's explore what affects sleep.
+
+### What is a circadian rhythm?
+
+It's a 24-hour internal clock, a regulator for sleepiness and wakefulness. When we feel the most energized can vary a bit from person to person, but for most adults, there is a noticeable drop in energy from 1 to 3 in the afternoon and from 2 to 4 at night.
+
+Outside factors like lightness and darkness can also have an impact on the internal clock. The brain sends a signal to the body to release melatonin which makes you sleepy.
+
+### Effects of sleep deprivation
+
+Human beings are the only species that deprive themselves of sleep and we are not wired to compensate for the lack of sleep. Sleep is not like a bank. We can't accumulate a debt and hope to pay it off on the weekend.
+
+![Airport sleep](./airport-sleep.jpg)
+
+Effects of sleep deprivation can show:
+
+- **Worsened brain performance**. Getting less than 6 hours of sleep will drop the time to physical exhaustion by 30%.
+
+- **Lower muscle strength**. The less sleep you have less muscle strength, as well as jump height and running speed decrease.
+
+- **Lack of testesterone**. Men who sleep only 5 to 6 hours a night will have a level of testosterone 6-10 years their senior.
+
+Long term lack of sleep is also connected to diseases, one of them is Alzheimer's disease. This may come as a shocker, but drowsy, sleep deprived driving kills more people on the roads than alcohol or drugs combined.
+
+### How to improve sleep?
+
+Here are some tips to help you with night's sleep:
+
+- **Keep a regular sleep schedule**. Go to bed at the same time every day, try to do it for weekends too. This way, you'll train the body to fall asleep at the same time.
+
+- **Avoid screen time before bed**. Blue light, emitted from mobile phones can postpone melatonin production, which is needed for the body to feel sleepy. One hour of screen time can postpone sleepiness for up to 3 hours.
+
+- **Cool, clean and quiet bedroom**. Bedroom temperature should be between 60 and 67 degrees (15 to 20 degrees Celsius). Eliminate any noises that can disturb sleep and blackout any sources of light from outside lamps or electronic devices.
+
+- **Eat 2 to 3 hours before bed**. This way, you won't be too hungry or too full going to bed.
+
+![Bedroom](./bedroom.jpg)
+
+### "There is an app for that."
+
+To measure the quality of your sleep, I recommend downloading an app called [Sleep as Android](https://play.google.com/store/apps/details?id=com.urbandroid.sleep), available for Android.
+
+It allows you to track sleep and use an ability to set up "smart" alarms, waking you up at the end of the sleep cycle. There are similar apps for iOS as well.
```

Comment:
_use an ability_

i'm not sure if you can say that sth/sb ''uses'' the ability. 'has' would be better here instead of 'use' (and yes, put it in the singular as it probably doesn't refer back to 'allows' but to 'it')

also, 'the' instead of 'an'